### DISCUSSION TITLE
Navigator examples?

### DISCUSSION THREAD

**Discussion by mikeymike9000** [hmlongco/Navigator#2] [General]
For those who don't have a paid medium account, is there anywhere to see visual examples of Navigator?

**Comment by hmlongco** [comment]
Friend link...

https://medium.com/@michaellong/now-previewing-navigator-faebf290a1da?sk=88d3ff52057cf0a948279e6be4a15f75

**Reply by mikeymike9000** [reply]
thanks Michael, I'll give it a whirl in the coming days and try to give some feedback
